bedrock/firefox/templates/firefox/new.html

```diff
@@ -118,12 +120,18 @@ <h1 class="large" id="main-headline">
         <div class="android download-button-wrapper" data-upgrade-subtitle="{{_('Get it free on Google Play')}}">
           {{ download_firefox(platform='android', small=True, icon=False, dom_id='download-button-android') }}
         </div>
+        <div class="ios download-button-wrapper">
+          {{ download_firefox(platform='ios') }}
```

Comment:
Note: this button doesn't work. That'll be a separate PR that will effectively block this one.